packages/common/src/types/conditionalFormatting.ts

```diff
@@ -13,7 +13,6 @@ export type ConditionalFormattingMinMax<T = number> = {
 export type ConditionalFormattingColorRange = {
     start: string;
     end: string;
-    steps: number;
```

Comment:
The idea here was to have a stepper slider in the UI at some point. Since the DB already has the values, and it’s not causing any issues, I’d just leave it.

Reply:
discussed internally. ignore.